packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/Controls.js

```diff
@@ -1,20 +1,43 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import {
   Box,
   CheckBox,
   FormField,
   RadioButtonGroup
 } from 'grommet'
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 import { PlainButton, SpacedText, withThemeContext } from '@zooniverse/react-components'
 import counterpart from 'counterpart'
 import FlipIcon from '../FlipIcon'
 import en from '../../locales/en'
 import theme from './theme'
-import { PropTypes } from 'mobx-react'
 
 counterpart.registerTranslations('en', en)
 
+export const StyledRadioButtonGroup = styled(RadioButtonGroup)`
+  > div {
+    position: relative;
+  }
+
+  > div:after {
+    ${props => css`border-bottom: 1px solid ${props.color};`}
+    bottom: 0;
+    content: '';
+    height: 5px;
+    left: 1px;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 28px;
+  }
+
+  > div:first-of-type:after {
+    left: -5px;
+    width: 34px;
+  }
```

Comment:
I realize these styles are goofy. I basically eye-balled it rather than try to hit an exact size. The distance between each radio button isn't exactly the same. I think because the label is slightly bigger for `0.5`.